roles/addons/diskless/readme.rst

```diff
@@ -122,16 +122,37 @@ Select the kernel you want to use wit this image from the list of available kern
  
 Choose a standard image, and give a size.
 
+Notes:
+
+* The size recommended for the Standard or Custom is 5G
+* The size of the final image booted and loaded in RAM is approximately : 1.2Gb for Standard image, 250Mb for Small image, 130 Mb for Minimall image and 1.2Gb for Custom image
```

Comment:
1.2Gb = 150MB.
In my tests yesterday, I was not able to get a 1.2GB file. It's either a 811MB squashfs or a 2.1GB rootfs image.

Reply:
1.2Gb = 150MB ?? 
Did you check the size of the image in the booted node?
with `free -h` command

Reply:
1 Byte = 8 bits.
No I did not boot the diskless image. However I expect the total size to be 2.1GB.

Reply:
I destroyed the test env... I will create a new image and check the memory consumption.

Reply:
1.2 GB well seen ;)

Reply:
> I destroyed the test env... I will create a new image and check the memory consumption.

ok keep me informed
Thanks

Reply:
Image:
```
  Image name: 		test
  Kernel version: 	vmlinuz-4.18.0-193.6.3.el8_2.x86_64
  Root password: 	root
  Image profile: 	1
  Image size: 		2100M
  SSH pubkey: 		/root/.ssh/id_rsa.pub
  Enable SELinux: 	y
```

Usage:
```
[root@c001 ~]# free -h
              total        used        free      shared  buff/cache   available
Mem:          1.9Gi       125Mi       877Mi       8.0Mi       982Mi       890Mi
Swap:            0B          0B          0B
[root@c001 ~]# df -h .
Filesystem           Size  Used Avail Use% Mounted on
/dev/mapper/live-rw  2.1G  1.4G  661M  69% /
```

Reply:
```
[root@pm5-nod40 ~]# free -h
              total        used        free      shared  buff/cache   available
Mem:           31Gi       **1.9Gi**        26Gi       121Mi       2.3Gi        27Gi
Swap:            0B          0B          0B
```

```
[root@pm5-nod40 ~]# df -h
Filesystem           Size  Used Avail Use% Mounted on
devtmpfs              16G     0   16G   0% /dev
tmpfs                 16G     0   16G   0% /dev/shm
tmpfs                 16G  122M   16G   1% /run
/dev/mapper/live-rw  **5.0G**  2.5G  2.6G  49% /
tmpfs                 16G     0   16G   0% /sys/fs/cgroup
tmpfs                3.2G     0  3.2G   0% /run/user/0
```

Reply:
Any additional packages selected with the custom image option? Otherwise I would expect the same size than option 1.

Reply:
yes i have added additionnal packages and running the computes playbook.
All this added 500MB over the initial size.

Reply:
OK. The size will depend on the list of additional packages, we cannot provide any hint here. Thanks for testing.